### PR TITLE
Use symbols for namespacing

### PR DESCRIPTION
**Pull request by HenriBeck**
#220

### CODE REVIEW

File: src/ns.js [hunk 1]
```diff
@@ -1,7 +1,17 @@
+const hasSymbol = typeof Symbol === 'function' && Symbol.for
+
 /**
  * Namespaces to avoid conflicts on the context.
  */
-export const jss = '64a55d578f856d258dc345b094a2a2b3'
-export const sheetsRegistry = 'd4bd0baacbc52bbd48bbb9eb24344ecd'
-export const managers = 'b768b78919504fba9de2c03545c5cd3a'
-export const sheetOptions = '6fc570d6bd61383819d0f9e7407c452d'
+export const jss = hasSymbol
```
Comment (kof) [inline src/ns.js]: does this lint? I thought I disabled multiline ternaries in linter?

File: src/ns.js [hunk 2]
```diff
@@ -1,7 +1,17 @@
+const hasSymbol = typeof Symbol === 'function' && Symbol.for
```
Comment (kof) [inline src/ns.js]: I would also rename ns.js to symbols.js and other "ns" variables for the same thing

File: src/ns.js (hunk 1 above)
Comment (HenriBeck) [reply src/ns.js]: I didn't get any lint errors

**Comment by kof** [comment]
uhh, just tried, all test fail …

**Comment by kof** [comment]
need to fix the CI…

**Comment by kof** [comment]
@HenriBeck do you know whats wrong with tests?

**Comment by HenriBeck** [comment]
@kof I see this error:  No usable sandbox! Update your kernel or see https://chromium.googlesource.com/chromium/src/+/master/docs/linux_suid_sandbox_development.md for more information on developing with the SUID sandbox. If you want to live dangerously and need an immediate workaround, you can try using --no-sandbox.

**Comment by kof** [comment]
I mean locally, if you run `npm test` I get a bunch of errors

**Comment by HenriBeck** [comment]
It has something to do with the symbols but I'm not quite sure what.

**Comment by HenriBeck** [comment]
It works in the browser and does what it should do: https://codesandbox.io/s/0mn0vyp6wv

**Comment by kof** [comment]
Yep, we still need to fix tests though :)

**Comment by HenriBeck** [comment]
I have a free day tomorrow and will try to fix the tests then

**Comment by kof** [comment]
cool, please try to fix tests with symbols first, they block releases, otherwise we need to remove them from master

**Pull request merged**